azure_functions_worker/mmap_handler/file_accessor.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
```

Comment:
In the __init__.py of this folder (for some reason I cannot leave comments there), write a description with pointers to the C# code and probably the issue/docu.